main.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+const theadInner = ["title", "name", "description"];
+
+class Element {
+  constructor(tag, innerText) {
+    this.tag = tag;
+    this.innertext = innerText;
+  }
+  render() {
+    const element = document.createElement(this.tag);
+
+    element.textContent = this.innertext;
+    return element;
+  }
+}
+
+function listOfUsers() {
+  const xhr = new XMLHttpRequest();
+
+  xhr.open("GET", "https://jsonplaceholder.typicode.com/users", false);
+  xhr.send();
+  try {
+    return JSON.parse(xhr.responseText);
+  } catch (e) {
+    throw new Error("invalid JSON");
+  }
+}
+
+function listOfTodo() {
+  const xhr = new XMLHttpRequest();
+
+  xhr.open("GET", "https://jsonplaceholder.typicode.com/todos", false);
+  xhr.send();
+  try {
+    return JSON.parse(xhr.responseText);
+  } catch (e) {
+    throw new Error("invalid JSON");
+  }
+}
+const users = listOfUsers().map(user => {
+  return {
+    id: user.id,
+    email: user.email,
+    name: user.name
+  };
+});
+
+const todo = listOfTodo().map(user => {
```

Comment:
it is not a user it is todo, also this map is unneed, it might make sense if you find users in the map and merge user data in todo